### PR TITLE
Yamato improvement

### PR DESCRIPTION
**Pull request by vincent-unity**
1. Remove .aar build script from yamato.
2. Update project at https://github.com/Unity-Technologies/com.unity.mobile.notifications/tree/master/project to include the java source files in the package. Then you can simply open it in android studio and make changes, without exporting to gradle project.

To QA: I verified the project https://github.com/Unity-Technologies/com.unity.mobile.notifications/tree/master/project on Windows, could you please verify it on Mac as well? Thanks.

### CODE REVIEW

**Comment by vincent-unity** [comment]
@lucianUnity3D FYI.

File: project/androidnotifications/build.gradle [hunk 1]
```diff
@@ -36,22 +36,4 @@ dependencies {
...
-        copy {
-            from('build/outputs/aar') {
-                include '*-release.aar'
-            }
-            into '../../com.unity.mobile.notifications/Runtime/Android/Plugins/Android'
-        }
-    }
-}
\ No newline at end of file
+android.sourceSets.main.java.srcDirs += '../../com.unity.mobile.notifications/Runtime/Android/Plugins/Android/';
```
[the remark this answers is not in the document]
Comment (vincent-unity) [reply project/androidnotifications/build.gradle]: With this, we can simply open the project at https://github.com/Unity-Technologies/com.unity.mobile.notifications/tree/master/project in Android Studio, and make changes to the  .java files in the plugin. No need to export again in unity.

[the remark this answers is not in the document]
Comment (todi1856) [reply project/androidnotifications/build.gradle]: Yes, but you cannot test this, so not sure if it's that usable. 

Anyways, minor issue

[the remark this answers is not in the document]
Comment (vincent-unity) [reply project/androidnotifications/build.gradle]: Actually previsouly I thought if we should keep running the `build_package.sh` script in a separate job to make sure those java files still compile.
But we might add tests to running the package on device later which means we will test build anyway, so this is what I was hesitating...

**Pull request merged**